sources/packages/backend/apps/db-migrations/src/sql/User/Add-index-first-name-last-name.sql

```diff
@@ -0,0 +1,5 @@
+-- # pg_trgm extension to enable gin index.

```

Comment:
Is there a reason for the "#" at the beginning of all comments?

Reply:
I just followed the pattern from other sql files. I can remove it. Only `--` is required for comment

Reply:
Removed the `#`